notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class TapasForQuestionAnswering(model.__class__):  # it is better to keep the class name to avoid warnings
</pre><p>Why didn't you import the class from transformers?&nbsp;This code is coupled with an instance of the model and cannot be used without it, so the next time someone tries to make a standalone gradio app it will fail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Changed